doc/api/deprecations.md

```diff
@@ -2655,6 +2655,20 @@ Type: Documentation-only
 The [`crypto.Certificate()` constructor][] is deprecated. Use
 [static methods of `crypto.Certificate()`][] instead.
 
+### DEP0XXX: `fs.rmdir(path, { recursive: true })`
+<!-- YAML
+changes:
+  - version: REPLACME
+    pr-url: https://github.com/nodejs/node/pull/35562
+    description: Runtime deprecation.
+-->
+
+Type: Documentation-only
+
+`fs.rmdir(path, { recursive: true })` will throw on nonexistent paths, or when
```

Comment:
The future tense here is... subtle. This isn't the behavior that it exhibits now but it kind of sounds like it is. "If you use this function this way, it will throw" rather than "It works right now, but it will throw eventually". Maybe this instead?

```suggestion
In future versions of Node.js, `fs.rmdir(path, { recursive: true })` will throw on nonexistent paths, or when
```

(If suggestion is accepted, will need to wrap at 80 chars for linting, I think.)

Reply:
I like this rewording, I will update accordingly in each place.